modules/server/src/projects/addType.js

```diff
@@ -7,7 +7,7 @@ import getIndexPrefix from '../utils/getIndexPrefix';
 export default async (req, res) => {
   let { es } = req.context;
   let { id } = req.params;
-  let { index, name } = req.body;
+  let { index, name, esType } = req.body;
```

Comment:
maybe add the `esType` fallback after the if statement below, `if (!esType) esType = index`